Source/AwsCommonRuntimeKit/io/retryer/CRTExponentialBackoffRetryOptions.swift

```diff
@@ -1,14 +1,14 @@
 //  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 //  SPDX-License-Identifier: Apache-2.0.
-
+import AwsCIo
 public typealias GenerateRandom = () -> UInt64
 
-public struct CRTExponentialBackoffRetryOptions {
-    let eventLoopGroup: EventLoopGroup
-    let maxRetries: Int
-    let backOffScaleFactor: UInt32
-    let generateRandom: GenerateRandom?
-    let jitterMode: CRTExponentialBackoffJitterMode
+public struct CRTExponentialBackoffRetryOptions: CStruct {
+    public var eventLoopGroup: EventLoopGroup
+    public var maxRetries: Int
+    public var backOffScaleFactor: UInt32
+    public var jitterMode: CRTExponentialBackoffJitterMode
+    public let generateRandom: GenerateRandom?
```

Comment:
It's a callback, so I haven't made it a var.

Reply:
Yeah, that makes sense. Thanks. I thought it might give the wrong perception that callback can be updated if it hasn’t been triggered yet. But now, this makes more sense as Struct are a value type. I will update it.